packages/flutter_tools/test/integration.shard/ios_content_validation_test.dart

```diff
@@ -94,6 +84,8 @@ void main() {
       });
 
       testWithoutContext('flutter build ios builds a valid app', () {
+        // Should only contain Flutter.framework and App.framework.
+        expect(frameworkDirectory.listSync().length, 2);
```

Comment:
This will detect future regressions that could cause https://github.com/flutter/flutter/issues/76958.